canonicalwebteam/blog/django/urls.py

```diff
@@ -49,6 +50,22 @@ def to_url(self, value):
     path(r"/<yyyy:year>/<str:slug>", article_redirect),
     path(r"/feed", feed),
     path(r"/latest-news", latest_news),
+    path(
+        r"/cloud-and-server", group, {"slug": "cloud-and-server"}, name="group"
```

Comment:
I feel rather like defining these specific URLs should be beyond the scope of this module, as it seems like a specific use-case. It feels wrong for this module to know anything about e.g. the "cloud-and-server" group specifically.

I think this module should provide the `group` view but that these URLs should be kept in www.ubuntu.com's `urls.py`.

If this module were to provide generic group pages, I would have thought they'd take the form `/groups/{name}`, but we probably shouldn't create that endpoint yet as we don't have a use-case for it.